src/extension.ts

```diff
@@ -93,7 +93,12 @@ export async function activateInternal(context: vscode.ExtensionContext, perfSta
             if (!node) {
                 node = await ext.tree.showTreeItemPicker(accountContextValues.map((val: string) => val += AttachedAccountSuffix), actionContext);
             }
-
+            if (node instanceof MongoAccountTreeItem) {
+                if (node.fullId === ext.connectedMongoDB.parent.fullId) {
```

Comment:
`ext.connectedMongoDB` could be undefined causing a null ref exception here

You have to be extra careful with things that could be undefined in this repo because we haven't turned `strictNullChecks` on yet (which would automatically check this stuff for us). That work is tracked by https://github.com/microsoft/vscode-cosmosdb/issues/1220